### DISCUSSION TITLE
CadQuery would be a lot easier to master if there were more examples

### DISCUSSION THREAD

**Discussion by gitterman** [CadQuery/cadquery#1601] [Ideas]
The cadquery examples are a good point to start, but I am missing more explanations (e.g. comments) on the examples.
Also, many examples are just for the basic functionality and the links which point to the class documentation should also show syntax examples.
The loft example for instance has a combine=True option without commenting it and the class definition talks about other options but in a very technical programmers' language which is hard to understand for many.
I think it would help a lot if there were example lines on different options to give users a better clue.
In my case, I am trying to subtract a loft from a cylinder to get some inner cone, but I just couldn't figure out how the syntax for it.

**Comment by adam-urbanczyk** [comment]
Thanks for the feedback, noted (#1602). 

Actually `combine=True` in the loft example is not really needed because the `combine` option is set to `True` by default. To achieve what you want you can use `combine="cut"`.
